tests/commands/test_new.py

```diff
@@ -5,28 +5,41 @@
 from onyo.lib import Repo
 import pytest
 
-directories = ['./',
-               'simple',
+directories = ['simple',
                's p a c e s',
                's p a/c e s',
                'r/e/c/u/r/s/i/v/e',
                'overlap/one',
                'overlap/two',
                'overlap/three',
+               'spe\"cial\\char\'acteஞrs',
                'very/very/very/deep'
                ]
 
-
-@pytest.mark.repo_dirs('simple',
-                       's p a c e s',
-                       's p a/c e s',
-                       'r/e/c/u/r/s/i/v/e',
-                       'overlap/one',
-                       'overlap/two',
-                       'overlap/three',
-                       'very/very/very/deep')
+@pytest.mark.repo_dirs(*directories)
 @pytest.mark.parametrize('directory', directories)
 def test_new(repo: Repo, directory: str) -> None:
+    """
+    Test that `onyo new` can create an asset in different directories.
+    """
+    file = f'{directory}/laptop_apple_macbookpro.0'
+    ret = subprocess.run(['onyo', 'new', '--yes', file], capture_output=True, text=True)
+
+    # verify correct output
+    assert "The following will be created:" in ret.stdout
+    assert file in ret.stdout
+    assert not ret.stderr
+    assert ret.returncode == 0
+    assert Path(file).exists()
+
+    # verify that the new asset exists and the repository is in a clean state
+    assert len(repo.assets) == 1
+    repo.fsck()
+
+
+@pytest.mark.repo_dirs(*directories)
+@pytest.mark.parametrize('directory', directories)
+def test_new_interactive(repo: Repo, directory: str) -> None:
     """
     Test that `onyo new` can create an asset in different directories.
```

Comment:
This docstring is identical to the one for `test_new()`, yet they focus on testing different things.